src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -89,11 +89,13 @@ class PowerDistributingActor:
     printed.
 
     Example:
-        ``` python
+        ```python
         import grpc.aio as grpcaio
 
-        from frequenz.sdk.microgrid.graph import _MicrogridComponentGraph
+        from frequenz.sdk.microgrid._graph import _MicrogridComponentGraph
```

Comment:
>  Doesn't pylint complains about unused imports?

It would, but I disabled hat as I wildcard import everything in the file as well as the original imports for the examples

Reply:
Though, thinking about this again, I suppose I could disable all those auito-imports with comments and keep it enabled in the "global" options

Reply:
I played around a bit and managed to enable a few more import warnings and thus fixed a few more examples.